Build/README.md

```diff
@@ -1,8 +1,3 @@
-Instructions
-==
-
-In general, you can follow the guides laid out for StepMania [here](https://github.com/stepmania/stepmania/wiki/Compiling-StepMania) as long as you replace references to stepmania with itgmania.
```

Comment:
I think having this reference is still nice (because I reference it pretty often). Ideally we could port the wiki, but for now linking SM's is totally fine. Maybe add a blurb that saying while everything won't be applicable to ITGmania, it's still a good resource.

Reply:
I rewrote it slightly:

> Most users will find everything they need in this README. There are differences in the build process for ITGmania as compared to Stepmania, so it is recommended to stick with these ITGmania instructions. For anything not covered in this guide, you can usually refer to the original Stepmania documents [here](https://github.com/stepmania/stepmania/wiki/Compiling-StepMania), as long as you replace references to Stepmania with ITGmania.